src/tools/playground/build.gradle.kts

```diff
@@ -0,0 +1,28 @@
+import com.github.gradle.node.npm.task.NpmTask
+
+plugins {
+    alias(libs.plugins.node.gradle.plugin)
```

Comment:
Let's start using this plugin in a follow up PR:

https://github.com/node-gradle/gradle-node-plugin/blob/main/docs/usage.md

https://github.com/flock-community/wirespec/issues/352